src/styles/Blocks/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    display: flex;
+    width: 16px;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+    background-image: url(../images/star.svg);
+  }
+}
+
+.stars--4 :nth-child(-n+4) {
```

Comment:
you need to move all modifiers from the previous task according to  the requirements
![image](https://github.com/mate-academy/layout_product-cards/assets/108538921/ba87046b-888e-4ec9-9829-67812e2f03aa)